src/index.ts

```diff
@@ -2,40 +2,63 @@ import { multinetAxiosInstance, MultinetAxiosInstance } from './axios';
 
 import axios, { AxiosRequestConfig } from 'axios';
 
+export interface Paginated<T> {
+  count: number,
+  next: string,
+  previous: string,
+  results: T[],
+}
+
+export interface Table {
+  name: string;
+  edge: boolean;
+  id: number;
+  created: string;
+  modified: string;
+  workspace: Workspace[];
+}
+
 export interface TableRow {
   _key: string;
   _id: string;
 }
 
-export interface GraphSpec {
-  edgeTable: string;
-  nodeTables: string[];
+export interface Graph {
+  id: number;
+  name: string;
+  created: string;
+  modified: string;
 }
 
-export interface NodesSpec {
-  count: number;
-  nodes: TableRow[];
+export interface GraphSpec {
+  id: number;
+  name: string;
+  node_count: number;
+  edge_count: number;
+  created: string;
+  modified: string;
+  workspace: Workspace;
 }
 
-export interface RowsSpec {
-  count: number;
-  rows: TableRow[];
-}
 
 export interface Edge {
-  edge: string;
   from: string;
   to: string;
 }
 
 export interface EdgesSpec {
-  count: number;
-  edges: Edge[];
+  _key: string;
+  _id: string;
+  _from: string;
+  _to: string;
+  _rev: string;
```

Comment:
I think if we're going to have `_rev` here, we should also add it to `TableRow`, for consistency, or remove it entirely.

Reply:
I went ahead and added it to `TableRow`.

See https://github.com/multinet-app/multinetjs/pull/42/commits/b3bc6b705d5dafbf44320c6a32e7bf139c4b7378#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R24